src/ConditionalRenderingComponent.jsx

```diff
@@ -0,0 +1,20 @@
+const Color = ({text, isBlue})=> {
+    return (
+        <span> {text}{isBlue ? 'blue' : 'red'} </span>
+        );
+  }
+
+  export function ConditionalRenderingComponent (){
+    return (
+    <span>
+        <Color
+        text= 'I am '
+        isBlue={true}
+        /> 
+        <Color 
+        text = 'I am '
+        isBlue={false}
+        /> 
+        </span>
+    )
+  }
```

Comment:
We want to only update the `ConditionalRenderingComponent` without adding new components. We also do not want to remove props from the component as described in AC - 2C.